### DISCUSSION TITLE
Downloading data for Ohio Courts only

### DISCUSSION THREAD

**Discussion by deantaylormax** [freelawproject/courtlistener#3201] [Q&A]
I am familiar with APIs and have a key for CourtListener.  Is there an option to download a zip file of the latest updates to the Ohio court's dataset?  I looked through the site for an hour or more and could not find any information on how to do this for just the state of Ohio.

**Comment by mlissner** [comment]
You can get bulk data by object type in a bz2 file, but you can't get anything for just a single court. Your best bet is to use the API and filter to just the correct states. If you look at https://github.com/freelawproject/courtlistener/discussions/3215, there are some tips for how to do that?
